notebooks/Periodicity-Detection/Periodicity_Detection.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>Remove me, this is your work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>